product_docs/docs/biganimal/release/using_cluster/05_monitoring_and_logging/health_status.mdx

```diff
@@ -0,0 +1,71 @@
+---
+title: "Health Status"
+---
+
+Health Status dashboard provides the status of the Postgres high-availability clusters. It provides status for both Primary/Standby and Distributed high-availability clusters.
+
+Health Status dashboard displays:
+- A set of cluster-wide health indicators that helps to draw your attention immediately to the critical issues.
+- A schematic view of all the nodes organized in a cluster distributed across regions displaying health and role(s) of each node.
+- A replication status view displaying the status of each replication slot and the associated replication lags.
+
+## Viewing Health Status dashboard
+
+To view the **Health Status** dashboard from the BigAnimal portal:
+
+1. In the left navigation of BigAnimal Portal, go to **Clusters**.
+2. Select any ready high-availability or PGD cluster.
+3. On the cluster detail page, select the **Health Status** tab.
+
+The **Health Status** tab displays the dashboard with health status categorized in sections:
+- [Viewing Health Status dashboard](#viewing-health-status-dashboard)
+  - [Global cluster health](#global-cluster-health)
+  - [Regional nodes health and roles](#regional-nodes-health-and-roles)
+  - [Replication status](#replication-status)
+
+### Global cluster health
+
+The global cluster health section displays the cluster-wide view including the following metrics:
+
+- **Raft Status** (PGD only)&mdash; It indicates whether the Raft consensus algorithm is running correctly in the cluster. It verifies that one node is elected as the global leader and the Raft roles such as RAFT_LEADER and RAFT_FOLLOWER are defined correctly across all the nodes.
+- **Replication Slot Status** (PGD only)&mdash; It indicates whether all the replication slots are in streaming state or not.
+- **Clock Skew**&mdash; It indicates whether the node's clock is in sync and doesn't exceed a threshold of 60 seconds.
+- **Proxy Status** (PGD only)&mdash; It provides the number of PGD proxies up and running as compared to the available proxies.
+- **Node Status**&mdash; It provides the number of nodes that are up and running.
+- **Transaction rate**&mdash; It provides the total number of transactions including committed and rolled back transactions per second in the cluster.
```

Comment:
On all of these lists, it would be more idiomatic (at least in American/British English) to omit the 'It' following the dash. Effectively the name of the label then functions as the subject of the sentence. We could even omit the dash and make this more explicit.